src/core/resources-templates.tf

```diff
@@ -0,0 +1,19 @@
+resource "null_resource" "upload_resources_templates" {
+  triggers = {
+    dir_sha1 = sha1(join("", [for f in fileset("${path.module}/resources_templates", "**") : filesha1("${path.module}/resources_templates/${f}")]))
```

Comment:
please can you bring a more clear name for variable "f"

Reply:
"f" is for "file", i have not a specific name to use, then i shortened it to just "f"
I don't think there is reason to use a long name here

Reply:
ok